src/eos-autoupdater.c

```diff
@@ -61,6 +63,7 @@ static const char *UPDATE_STAMP_DIR = LOCALSTATEDIR "/eos-updater";
 static const char *UPDATE_STAMP_NAME = "eos-updater-stamp";
 
 static const char *CONFIG_FILE_PATH = SYSCONFDIR "/eos-updater.conf";
+static const char *STATIC_CONFIG_FILE_PATH = DATADIR "/" PACKAGE_NAME "/etc/eos-updater.conf";
```

Comment:
Bikeshedding - I don't think the additional `etc/` subdir is necessary. There's nothing in the pkgdatadir right now, and I can't see us accumulating enough files where the additional subdirectory is useful for disambiguation.